Makefile

```diff
@@ -25,7 +25,7 @@ cert: ## [HTTP] Generate self signed certificate
 
 # DOCKER TASKS
 docker-build: ## [DOCKER] Build given container. Example: `make docker-build BIN=user`
-	docker build -f cmd/$(BIN)/Dockerfile --no-cache --build-arg BIN=$(BIN) --build-arg VERSION=$(VERSION) --build-arg GIT_COMMIT=$(GIT_COMMIT) -t $(BIN) .
+	docker build -f cmd/$(BIN)/Dockerfile --no-cache --build-arg BIN=$(BIN) --build-arg VERSION=$(VERSION) --build-arg GIT_COMMIT=$(GIT_COMMIT) -t $(BIN):local .
```

Comment:
make file should work with production env,
should not be preset to local env

maybe we need to add an option to set a tag and then update doc examples with `local` tag